### DISCUSSION TITLE
How to switch focus to previous window in the same workspace?

### DISCUSSION THREAD

**Discussion by TTtrace** [hyprwm/Hyprland#2037] [Q&A]
The `focuscurrentorlast` dispatcher will move focus to the previously focused window, no matter in which workspace. 

I hope to use `ALT + Tab` to switch focus between two windows in the same workspace, how to achieve that?

Further more, if we type `ALT + Tab ` once, focus will cycle through just two window in the workspace; if we type `ALT + Tab ` many times, focus can cycle through all the windows in the workspace. This behavior looks like in Windows/macOS. Is this possible?

**Comment by dggavancho** [comment]
You can achive that by adding the following line to hyprland.conf

bind = $mainMod, TAB, cyclenext,

Also, see https://wiki.hypr.land/Configuring/Dispatchers/ for more reference.

**Reply by ultimateBroK** [reply]
it's now focuscurrentorlast
